com/TestSources/PyCOMTest/PyCOMTest.idl

```diff
@@ -70,7 +70,7 @@ library PyCOMTestLib
 		const long LongTest2 = 0x7FFFFFFFL;
 		const unsigned char UCharTest = 255;
 		const char CharTest = -1;
-		const LPWSTR StringTest = L"Hello Wo�ld";
```

Comment:
Looks like you accidentally opened/saved this file with a different encoding!

This is one of two files that can't be in UTF-8. Depending on your editor, try reopening as `windows1252` (aka `ISO-1252`, aka `Western (Windows 1252)`), or as `latin-1`. Then correct this and the `©` character, then save using the same encoding.